### PR TITLE
Write swagger doc for ticket ordering. Model creation

### PR DESCRIPTION
**Pull request by sobchenko**

### CODE REVIEW

File: doc/theatre.json [hunk 1]
```diff
@@ -3,25 +3,27 @@
     "info": {
         "version": "2.0.0",
         "title": "Cherkasy Drama Theatre",
-        "termsOfService": "http://theatre-shevchenko.ck.ua/en//terms/",
+        "termsOfService": "http://theatre-shevchenko.ck.ua/en/terms/",
         "contact": {
-            "name": "GeekHub Foundation",
```
Comment (spolischook) [inline doc/theatre.json]: revert this contact changes

File: doc/theatre.json [hunk 2]
```diff
@@ -3,25 +3,27 @@
...
-            "email": "neek@geekhub.ck.ua"
+            "name": "neek@geekhub.ck.ua"
         },
         "license": {
             "name": "Apache 2.0",
             "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
         }
     },
-    "basePath": "/",
+    "host": "api.theatre.pp.ua",
```
Comment (spolischook) [inline doc/theatre.json]: remove host

File: doc/theatre.json [hunk 3]
```diff
@@ -147,7 +155,7 @@
                         "description": "Slug of employee in database",
                         "required": true,
                         "type": "string",
-                        "pattern": "/^[a-z\\d-]+$/"
```
Comment (spolischook) [inline doc/theatre.json]: Revert changes of pattern, cause tests will not pass

File: doc/theatre.json [hunk 4]
```diff
@@ -251,7 +265,10 @@
         },
         "/posts/{slug}": {
             "get": {
-                "operationId": "get_post",
+                "tags": [
+                    "posts"
+                ],
+                "operationId": "findPostBySlug",
```
Comment (spolischook) [inline doc/theatre.json]: "operationId": "get_post"

File: doc/theatre.json [hunk 5]
```diff
@@ -542,7 +482,10 @@
         },
         "/performanceevents": {
             "get": {
-                "operationId": "get_performanceevents",
```
Comment (spolischook) [inline doc/theatre.json]: take back this

**Comment by sobchenko** [comment]
Changes already in master branch. Not necessary anymore.

**Pull request closed without merging**